pkg/platform/coverage/golang/golang.go

```diff
@@ -81,14 +90,14 @@ func (g *Golang) GetCoverage() (models.TestCoverage, error) {
 		return testCov, err
 	}
 
-	generateCovTxtCmd := exec.CommandContext(g.ctx, "go", "tool", "covdata", "textfmt", "-i="+os.Getenv("GOCOVERDIR"), "-o="+os.Getenv("GOCOVERDIR")+"/total-coverage.txt")
+	generateCovTxtCmd := exec.CommandContext(g.ctx, "/usr/local/go/bin/go", "tool", "covdata", "textfmt", "-i="+coverageDir, "-o="+coverageDir+"/total-coverage.txt")
```

Comment:
Why we are providing path to go binary in the new change ??

Reply:
reverted it by adding go binary path in /etc/sudoers through dockerfile